src/pint/models/timing_model.py

```diff
@@ -2223,7 +2229,7 @@ def remove_param(self, param):
             param_name = param.name
         if param_name not in self.params:
             raise ValueError(
-                "Tried to remove parameter {} but it is not listed: {}".formmat(
+                "Tried to remove parameter {} but it is not listed: {}".format(
```

Comment:
If you're changing this might as well use f-strings.

Reply:
done.